Sonobi - Add Video Docs

JonGoSonobi opened the pull request:
Added example html to show an example implementation of sonobi video

JonGoSonobi commented:
@mkendall07 can we get this one merged as well?

bretg commented:
I removed the change to bidders.md as this is now dynamic. Please add the following to sonobi.md
```
media_types: video
```
Also - the test page that you've embedded in the doc is quite large. Is there any way you can place that code somewhere else and link to it?

@mkendall07 -- is there a better place for bidders to add this kind of example? It's a pretty nice full-page example for video, but really not much different than what's in http://prebid.org/dev-docs/show-video-with-a-dfp-video-tag.html. Maybe there's something there we can replace?

@JonGoSonobi - is it true that the only thing unique to Sonobi in the example is which video parameters are supported?
```
             video: {
                skipppable: false,
                playback_methods: ['auto_play_sound_off']
              }
```

JonGoSonobi commented:
@bretg I added the media_types: video.

As far as the video parameters in the example videoAdUnit, that was just left over from the example page I used to test from the prebid.js project at test/pages/video.html. I removed this since it isn't used.

We wanted to provide a working example page of our video implementation. Is there another section in the prebid docs where I can put this example for Sonobi Video and link to it from the bidder params doc?

bretg commented:
How about making a new directory under https://github.com/prebid/prebid.github.io/tree/master/examples -- say 'bidders'  and placing your file there as something like 'sonobi-video-example.html'.

JonGoSonobi commented:
Sounds good. I moved it there.

bretg commented:
Thanks @JonGoSonobi - merging, as 1.8 is released.
